moon/cfc_77.15.580/server/net.moon

```diff
@@ -59,14 +87,16 @@ net.Incoming = ( len, client ) ->
         plyNick = client\Nick!
         plyIP = client\IPAddress!
 
-    tallyUsage lowerStr, client, plySteamId, plyNick, plyIP
+    shouldIgnore = tallyUsage lowerStr, client, plySteamId, plyNick, plyIP
+    return unless shouldIgnore ~= nil
```

Comment:
why not just `return if shouldIgnore`?

Reply:
Wasn't sure what I wanted `tallyUsage` to return